core/Network.js

```diff
@@ -97,11 +97,16 @@ Network.prototype.defProc = function (func, name) {
   if (!func) {
     throw new Error("No function passed to defProc");
   }
-  var proc = new Process(name || func.name, func);
-  trace('Created Process with name: ' + proc.name);
-
-  this._processes.push(proc);
-  return proc;
+  var s = name;
+  if (s == null)
+	  s = func.name;
```

Comment:
The problem with this is that func.name will not always be defined.

JavaScript supports anonymous functions:

`var foo = function () {}; // foo.name === undefined`

If you're going to use a process's name as the key in an associative array (which I think is a good idea), then we need an approach for default names.